src/metabase/driver/sql_jdbc/actions.clj

```diff
@@ -220,7 +220,7 @@
                             (tru "Sorry, the row you''re trying to delete doesn''t exist")
                             (tru "Sorry, this would delete {0} rows, but you can only act on 1" rows-deleted))
                           {:staus-code 400})))
-        {:rows-deleted [1]}))))
```

Comment:
Really curious how this ended up as a list in the first place, so weird 😄

Reply:
+1, especially given that bulk actions return it as a scalar